### PR TITLE
Preparation for static dispatch

### PR DESCRIPTION
**Pull request by calebzulawski**

### CODE REVIEW

**Comment by calebzulawski** [comment]
@TethysSvensson if you have no objections I'll pull this in

File: src/target_clones.rs [hunk 1]
```diff
@@ -63,34 +88,21 @@ impl ToTokens for FunctionClone<'_> {
...
-                #target_arch
-                #[inline(always)]
-                #recursion_helper_signature {
-                    unsafe {
-                        #outer_function_ident(#(#argument_names),*)
-                    }
-                }
-
-                #target_arch
+            pub #outer_signature {
```
Comment (TethysSvensson) [inline src/target_clones.rs]: I think this should probably be `max(outer_function_publicness, "pub(super)")`

Comment (TethysSvensson) [reply src/target_clones.rs]: I'm not sure it matters though.

**Comment by TethysSvensson** [comment]
I have not done a thorough code review, but I have looked at the generated output using `cargo expand`. That part looks good though :+1:

File: src/target_clones.rs (hunk 1 above)
Comment (calebzulawski) [reply src/target_clones.rs]: I think that would prevent static dispatching across modules, right?  I'm not sure that's desirable.  I'm not even opposed to someone making the generated functions crate-public (though I wouldn't do it)

**Comment by calebzulawski** [comment]
Good enough for me!

**Pull request merged**